pipelines/deployer/deployer.yaml

```diff
@@ -490,6 +490,35 @@ drain_cluster_task: &drain_cluster_task
       done
   inputs:
   - name: platform
+delete_svcop_resources_task: &delete_svcop_resources
```

Comment:
does this really need to be it's own task? ... feels like it could be part of the main drain task and avoid a bunch of duplication